config/routes.rb

```diff
@@ -1,4 +1,7 @@
 Rails.application.routes.draw do
+  devise_for :users, only: :omniauth_callbacks,
+      controllers: {omniauth_callbacks: "omniauth_callbacks"}
+
```

Comment:
Sao nó nằm ngoài I18n ri em, mà code này có lặp với code line 15 ko. Line 15 còn đang dùng nháy đơn nữa

Reply:
em để trong trong I18n thì bị lỗi Devise not support scoping á chị OmniAuth callbacks, còn line 15 em skip omniauth_callbacks rồi á chị